packages/web-components/package.json

```
@@ -191,7 +192,7 @@
     "start": "yarn start-storybook -p 6006 --docs --no-manager-cache",
     "start-storybook": "node node_modules/@storybook/html/bin/index",
     "build-storybook": "node node_modules/@storybook/html/bin/build -o ./dist/storybook --docs",
-    "e2e": "playwright test",
+    "e2e": "playwright test && npm run exports:check",
```

we don't use `npm`, also wondering why does this need to have separate alias ? within monorepo we wanna stick to common vocabulary for task targets in general, where another artificial one might not be worth it.

```suggestion
    "e2e": "playwright test && yarn exports:check",
```

---

shouldn't this be rather part of `build` ? that's the only command that creates actual assets matching public API surface.